Gemfile

```diff
@@ -18,7 +18,7 @@ gem 'jquery-rails', '>= 4.4.0'
 gem 'json-jwt', '>= 1.13.0'
 gem 'jwt'
 gem 'kaminari'
-gem 'lograge'
+gem 'lograge', '>= 0.11.2'
```

Comment:
We shouldn't depend on this gem directly, we get it from `identity-logging` --- I will remove it from here
https://github.com/18F/identity-logging/blob/main/identity-logging.gemspec#L26

Reply:
b66810c0ac8904ad8f609007d236293b725bf529